pkg/users/accounts-list.js

```diff
@@ -183,17 +236,82 @@
     });
 };
 
-const AccountsList = ({ accountsInfo, current_user, groups }) => {
-    const [currentTextFilter, setCurrentTextFilter] = useState("");
+const GroupsList = ({ groups, accounts }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+    const columns = [
+        { title: _("Group name"), sortable: true },
+        { title: _("ID"), sortable: true },
+        { title: _("# of users"), sortable: true },
+        { title: _("Accounts") },
+    ];
 
-    const accounts = mapGroupsToAccount(accountsInfo, groups);
+    const sortRows = (rows, direction, idx) => {
+        // GID and members columns are numeric
+        const isNumeric = idx == 1 || idx == 2;
+        const sortedRows = rows.sort((a, b) => {
+            const aitem = a.columns[idx].sortKey || a.columns[idx].title;
+            const bitem = b.columns[idx].sortKey || b.columns[idx].title;
+            const aname = a.columns[0].sortKey;
+            const bname = b.columns[0].sortKey;
 
-    const filtered_accounts = accounts.filter(account => {
-        if ((account.uid < 1000 && account.uid !== 0) ||
-                 account.shell.match(/^(\/usr)?\/sbin\/nologin/) ||
-                 account.shell === '/bin/false')
-            return false;
+            // administrator groups are always first
+            if (admins.includes(aname))
+                return direction === SortByDirection.asc ? -1 : 1;
+            if (admins.includes(bname))
+                return direction === SortByDirection.asc ? 1 : -1;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221206-122041-ea567d53-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)